console/typescript/docs_compiler.ts

```diff
@@ -7,8 +7,8 @@ import { BufReader } from "../../deps.ts";
 
 export default class ClassCompiler {
 
-  protected path: string;
-  protected process: string;
+  protected path: string = "";
```

Comment:
why initialize this with an empty string?

Reply:
i ran into `Property 'path' has no initializer and is not definitely assigned in the constructor.`. i changed the logic: 12c96d79ad7ad116558ee247b45dbd0009924d0e

Reply:
@saragee3 can you finish your review please?